index.php

```diff
@@ -72,7 +72,9 @@
 
                     <label class="checkbox">
                         <!--добавить сюда атрибут "checked", если переменная $show_complete_tasks равна единице-->
-                        <input class="checkbox__input visually-hidden show_completed" type="checkbox">
+                        <input class="checkbox__input visually-hidden show_completed"
+                               <?php if ($show_complete_tasks==1) echo " checked "   ?>
```

Comment:
Так, конечно, будет работать, но внутри html кода следует всегда использовать полную альтернативную запись if/endif
Конкретно в этом случае, можно сделать вот так:
```suggestion
                               <?= $show_complete_tasks==1 ? " checked " : '' ?>
```